x-pack/docs/en/security/authentication/jwt-realm.asciidoc

```diff
@@ -12,35 +12,61 @@ between the _client_ that is connecting to {es}, and the _user_ on whose behalf
 the request should run. The JWT authenticates the user, and a separate credential
 authenticates the client.
 
-A common scenario for JWTs is when an existing front-end application uses
-OpenID Connect (OIDC) to authenticate and identify a user, and then accesses {es}
-on behalf of the authenticated user.
-
-TIP: If the front-end application does not exist, you can use the
-<<token-authentication-services>> instead.
+The JWT realm is designed to cater for the following two scenarios:
+
+1. An application authenticates and identifies a user with an authentication flow, e.g.
+OpenID Connect (OIDC), and then accesses {es} on behalf of the authenticated user.
+A common scenario for JWTs is when an existing application uses
+2. An application itself goes through an authentication flow, e.g. OAuth2 Client Credentials
+Flow, and then accesses {es} for itself.
```

Comment:
I don't think this is super-clear. This might be a case where a few extra lines will help rather than hinder.

Perhaps:
```
2. An application accesses {es} using its own identity, encoded as a JWT, e.g. The application
authenticates itself to a central identity platform using an OAuth2 Client Credentials Flow and 
then uses the resulting JWT-based access-token to connect to {es}
```

Reply:
That said, I wonder whether we need a mini-section on how to use JWTs for this purpose (we have 1 for OIDC based flows).
We probably need to mention that OAuth2 doesn't always use JWT, and that you need to check with your identity platform to see what format it uses for access tokens.
It might also provide a place where we can explain why we enforce certain subjects at the realm level.
We seem to have a good amount of content explaining the details of settings, etc, but not enough that captures the intent / purpose of the access_token JWT realm type.

Reply:
I adopted your version and also added a mini section in parallel to the oidc one as suggested.